app/Console/Commands/InstallCommand.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class InstallCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'council:install';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Simplify installation process';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $this->intro();
+
+        if (!file_exists('.env')) {
+            exec('mv .env.example .env');
+            $this->line("\r\n.env file successfully created\r\n");
+        }
+        
+        exec('composer install');
+        $this->line("\r\n ~ App dependencies successfully installed\r\n");
+
+        exec('php artisan key:generate');
+        $this->line("\r\n ~ Secret key properly generated\r\n");
+
+        // TODO: handle different db types (SQLite etc..), currently defaults to MySQL
+
+        $dbEnv['DB_DATABASE'] = $this->ask('Database name');
+        $dbEnv['DB_USERNAME'] = $this->ask('Database user');
+        $dbEnv['DB_PASSWORD'] = $this->secret('Database password ("null" for no password)');
+
```

Comment:
Maybe we can add the host for the database to? A database can be somewhere else to when the application is in production.

This application is not just for development purposes.

Reply:
Yes, absolutely :)